com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Bob.java

```diff
@@ -464,7 +464,8 @@ private static Options getCommandLineOptions() {
         addOption(options, null, "defoldsdk", true, "What version of the defold sdk (sha1) to use", true);
         addOption(options, null, "binary-output", true, "Location where built engine binary will be placed. Default is \"<build-output>/<platform>/\"", true);
 
-        addOption(options, null, "use-vanilla-lua", false, "Only ships vanilla source code (i.e. no byte code)", true);
+        addOption(options, null, "use-vanilla-lua", false, "DEPRECATED! Use --use-lua-source instead.", true);
+        addOption(options, null, "use-lua-source", false, "Use uncompressed and unencrypted Lua source code instead of byte code", true);
```

Comment:
This is more representative of what actually happens. We bundle with Lua source code. It is not in any way connected to which Lua version we use.